webCompression.ts: additional type errors on pipeThrough and Response lines

## Problem

v0.14.2 fixed the `CompressionFormat` error (line 131) by adding `@ts-ignore`, but two more type errors remain on lines 132 and 139 of `webCompression.ts`:

```
webCompression.ts(132,44): error TS2345: Argument of type 'CompressionStream' is not assignable
  to parameter of type 'ReadableWritablePair<Uint8Array<ArrayBuffer> | ...>'.
  Types of property 'readable' are incompatible.
    Type 'ReadableStream<NonSharedUint8Array>' is missing the following properties from type
    'ReadableStream<...>': blob, text, bytes, json

webCompression.ts(139,23): error TS2345: Argument of type 'ReadableStream<...>' is not
  assignable to parameter of type 'BodyInit | null | undefined'.
  Type 'ReadableStream<...>' is not assignable to type 'ReadableStream<any>'.
    The types returned by 'getReader(...).read(...)' are incompatible between these types.
```

These fire when `tsc` runs with `@types/bun` (backend, CLI) because Bun's `ReadableStream` type adds `.blob()`, `.text()`, `.bytes()`, `.json()` methods that the Node/DOM `ReadableStream` type doesn't have.

## Current code (v0.14.2, lines 128–143)

```typescript
const format: Bun.CompressionFormat = encoding === "br" ? "brotli" : "gzip";
// @ts-ignore Bun supports "brotli" as CompressionFormat but DOM lib does not
const compressionStream = new CompressionStream(format);
const stream = response.body.pipeThrough(compressionStream);  // ← line 132

const headers = new Headers(response.headers);
headers.set("Content-Encoding", encoding);
headers.append("Vary", "Accept-Encoding");
headers.delete("Content-Length");

return new Response(stream, {  // ← line 139
  status: response.status,
  headers,
});
```

## Suggested fix

Add `@ts-ignore` before lines 132 and 139:

```typescript
// @ts-ignore Bun's ReadableStream type is incompatible with Node/DOM ReadableStream
const stream = response.body.pipeThrough(compressionStream);
...
// @ts-ignore Bun's ReadableStream type is incompatible with Node/DOM ReadableStream
return new Response(stream, {
```

Or cast via `as any` to suppress both in one shot.

## Affected consumers

Backend and CLI lint (`tsc` with `@types/bun`) both fail. Frontend lint (`tsc -b` with DOM lib) passes.

--- a/packages/keryx/package.json
+++ b/packages/keryx/package.json
@@ -1,6 +1,6 @@
 {
   "name": "keryx",
-  "version": "0.14.2",
+  "version": "0.14.3",
   "module": "index.ts",
   "type": "module",
   "license": "MIT",

--- a/packages/keryx/util/webCompression.ts
+++ b/packages/keryx/util/webCompression.ts
@@ -112,6 +112,7 @@ export async function compressResponse(
     const format: Bun.CompressionFormat = encoding === "br" ? "brotli" : "gzip";
     // @ts-ignore Bun supports "brotli" as CompressionFormat but DOM lib does not
     const compressionStream = new CompressionStream(format);
+    // @ts-ignore Bun's ReadableStream type is incompatible with Node/DOM ReadableStream
     const stream = new Blob([body]).stream().pipeThrough(compressionStream);
 
     const headers = new Headers(response.headers);
@@ -119,6 +120,7 @@ export async function compressResponse(
     headers.append("Vary", "Accept-Encoding");
     headers.delete("Content-Length");
 
+    // @ts-ignore Bun's ReadableStream type is incompatible with Node/DOM ReadableStream
     return new Response(stream, {
       status: response.status,
       headers,
@@ -129,6 +131,7 @@ export async function compressResponse(
   const format: Bun.CompressionFormat = encoding === "br" ? "brotli" : "gzip";
   // @ts-ignore Bun supports "brotli" as CompressionFormat but DOM lib does not
   const compressionStream = new CompressionStream(format);
+  // @ts-ignore Bun's ReadableStream type is incompatible with Node/DOM ReadableStream
   const stream = response.body.pipeThrough(compressionStream);
 
   const headers = new Headers(response.headers);
@@ -136,6 +139,7 @@ export async function compressResponse(
   headers.append("Vary", "Accept-Encoding");
   headers.delete("Content-Length");
 
+  // @ts-ignore Bun's ReadableStream type is incompatible with Node/DOM ReadableStream
   return new Response(stream, {
     status: response.status,
     headers,

